tests/api/cars/carsDomain.spec.js

```diff
@@ -0,0 +1,228 @@
+import { myFixture as test, expect } from "../../../src/fixtures/myFixture";
+import { CARS_DATA } from "../../../src/utils/carBrands&Models.js";
+
+test.describe("CarsController tests", () => {
+
+    test("should get all car brands", async ({ carsController }) => {
+
+        const response = await carsController.getBrands()
+
+        expect(response.status()).toBe(200)
+
+        const responseBody = await response.json()
+        expect(responseBody.status).toBe("ok")
+
+        //console.log(responseBody)
+
+        const expectedData = Object.values(CARS_DATA).map(brand => ({
+            id: brand.id,
+            title: brand.title,
+            logoFilename: brand.logoFilename,
+        }))
+
+        expect(responseBody.data).toEqual(expectedData)
+    })
+
+    test("should get car brand by ID", async ({ carsController }) => {
+
+        const brandId = CARS_DATA.BMW.id
+        const expectedBrand = {
+            id: brandId,
+            title: CARS_DATA.BMW.title,
+            logoFilename: CARS_DATA.BMW.logoFilename,
+        }
+
+        const response = await carsController.getBrandById(brandId)
+
+        expect(response.status()).toBe(200)
+
+        const responseBody = await response.json()
+        expect(responseBody.status).toBe("ok")
+
+        //console.log(responseBody)
+
+        expect(responseBody.data).toEqual(expectedBrand)
+    })
+
+    test("should get all car models", async ({ carsController }) => {
+
+        const response = await carsController.getModels()
+
+        expect(response.status()).toBe(200)
+
+        const expectedData = Object.values(CARS_DATA).flatMap(brand =>
+            Object.values(brand.model).map(model => ({
+                id: model.id,
+                carBrandId: model.carBrandId,
+                title: model.title
+            }))
+        )
+
+        const responseBody = await response.json()
+        expect(responseBody.status).toBe("ok")
+
+        //console.log(responseBody)
+
+        expect(responseBody.data).toEqual(expectedData)
+    })
+
+    test("should get car model by ID", async ({ carsController }) => {
+
+        const modelId = CARS_DATA.AUDI.model.TT.id
+        const expectedModel = {
+            id: modelId,
+            carBrandId: CARS_DATA.AUDI.id,
+            title: CARS_DATA.AUDI.model.TT.title,
+        }
+
+        const response = await carsController.getModelById(modelId)
+
+        expect(response.status()).toBe(200)
+
+        const responseBody = await response.json()
+        expect(responseBody.status).toBe("ok")
+
+        //console.log(responseBody)
+
+        expect(responseBody.data).toEqual(expectedModel)
+    })
+
+    test("should get all cars for a valid user", async ({ newUser, multipleCars, carsController }) => {
```

Comment:
first of all

you created all this cars by old user

and you fetch it by old user

you do not use new user anywhere